packages/libs/components/src/map/MapVEuMap.tsx

```diff
@@ -303,6 +313,9 @@ interface MapVEuMapEventsProps {
   onBoundsChanged: (bondsViewport: BoundsViewport) => void;
   onBaseLayerChanged?: (newBaseLayer: BaseLayerChoice) => void;
   setSelectedMarkers?: React.Dispatch<React.SetStateAction<string[]>>;
+  onMapClick?: (value: boolean) => void;
+  onMapDrag?: (value: boolean) => void;
+  onMapZoom?: (value: boolean) => void;
```

Comment:
Let's keep these generic and call them with any arguments. In other uses cases, it wouldn't be clear what the boolean `value` means.

```suggestion
  onMapClick?: () => void;
  onMapDrag?: () => void;
  onMapZoom?: () => void;
```